Makefile

```diff
@@ -0,0 +1,12 @@
+dev.firefox:
```

Comment:
can't we just use the scripts part of npm with package.json for these commands instead? Is the makefile that handy or common with web extension developers?

Reply:
We can indeed.

I got into the habit of using makefiles as a lowest common denominator, so that you don't need to know too much about each package manager for a project, on multi-language projects. I might remember npm, for example, but I've likely forgot the same kinds of steps for composer, and that's before we think about projects in Python, or Go, and how they handle these common tasks.

Does that clear it up a bit?

Reply:
If we can i would move that stuff into npm and then have the makefile as a front for it, so if you don't understand make you don't need to use it. 

So the same way that things can work with composer only, but with a makefile you can use make if you want. 
How does that sound?

Reply:
Sounds good. i've pushed the changes that now do this.